src/main/webapp/src/app/app.component.html

```diff
@@ -37,6 +37,6 @@ <h4 class="text-center">David Ladd</h4>
   <router-outlet></router-outlet>
 </div>
 <!-- Error homepage -->
```

Comment:
nit: Please update comment since this is not an error page anymore.

Reply:
Done